README.md

```diff
@@ -253,6 +254,37 @@ this.transitionToExternal('settings');
 
 For further documentation on this subject, view the [Engine Linking RFC](https://github.com/emberjs/rfcs/pull/122).
 
+### Accessing Engine Configuration Settings
+
+As in an application, you can provide configuration settings for your
+engine in `addon/config/environment.js`. You can access these settings in a
```

Comment:
I believe this path is off, it should be `<engine-root>/config/environment.js` (which you then import as if it were located in `addon/config/environment.js`).